Simulator/startup.resc

```diff
@@ -0,0 +1,93 @@
+$bin?=@Objects/controls-leader.elf
+$bps_bin?=@Objects/car-sim.elf
+$motor_bin?=@Objects/motor-sim.elf
+$platform=@Simulator/stm32f413.repl
+
+macro reset
+"""
+    sysbus LoadELF $bin
```

Comment:
does reset actually reset the emulation by just loading in the elf again? I would think that `Clear` does the same